javascript/javascript2/week2/indexhyfBayHelpers.html

```diff
@@ -0,0 +1,79 @@
+<head>
+  <title>HyfBay</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="main.css" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+    rel="stylesheet"
+  />
+</head>
+<body>
+  <main>
+    <header>
+      <div class="wrapper">
+        <div class="logo">
+          <img
+            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQMM3v5Fo-tWBnMyITJOPtyIUujfxmKM7jeZQ&usqp=CAU"
+            height="70px"
+            width="70px"
+            alt="logo"
+          />
+        </div>
+        <h1>HyfBay</h1>
+        <h2>- Buy the okay'est products</h2>
```

Comment:
Very good solution! Looks good, and does what it's supposed to.

One thing did confuse me a little but - maybe it would make sense to show all products when nothing is typed in the search bar?